docs/content/comparisons/druid-vs-storage-formats.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: doc_page
+---
+
+Druid vs Storage Formats (Parquet/Kudu)
+=======================================
+
```

Comment:
@toddlipcon I added some basic information about what I know but would love more feedback for a fair comparison and a discussion about tradeoffs. Any information you can add here would definitely help.